src/effects/VST3/VST3Utils.cpp

```diff
@@ -73,3 +76,70 @@ bool VST3Utils::ParseAutomationParameterKey(const wxString& key, Steinberg::Vst:
    return false;
 
 }
+
+
+
+Steinberg::IPtr<PresetsBufferStream> PresetsBufferStream::fromString(const wxString& str)
+{
+   Steinberg::Buffer buffer(str.size() / 4 * 3);
+   auto len = Base64::Decode(str, buffer);
+   assert(len <= buffer.getSize());
```

Comment:
You didn't write this assertion -- you only move a line that existed in another file.

But read this uncommented assertion and ask, what's the proof?

The proof should ideally be deduced from a postcondition stated explicitly in a comment in class Base64.

And size_t would be a better return type for `Decode`.

Reply:
I interpret your comment in this way:

1) remove the assertion, and write a comment in class Base64 that the returned value should be <= buffer.getSize()
2) size_t len = static_cast<size_t>( Base64::Decode(str, buffer) );

or do you mean
2) rewrite Base64::Decode so that it returns a size_t instead of an int ?